### PR TITLE
set annotations, provenance, PEP 8 compliance, Dockerfile cleanup

### PR DESCRIPTION
**Pull request by philerooski**

### CODE REVIEW

File: down_sample_re_count.py [hunk 1]
```diff
@@ -1,47 +1,61 @@
...
-    return(str2)
+    str2 = ''.join([fileName, '_counts.txt'])
+    return str2
 
-def push_to_synapse(fileName,uploadLocation):
-    syn = synapseclient.Synapse()
-    syn.login(os.environ['synapseUser'],os.environ['synapsePassword'])
-    entity = synapseclient.File(fileName,parentId = uploadLocation)
+
+def push_to_synapse(fileName, uploadLocation, synId, annotations, syn):
```
Comment (blogsdon) [inline down_sample_re_count.py]: do you set annotations in this function?

**Review by blogsdon — COMMENTED** [review]
make sure that annotations are set on the file

File: down_sample_re_count.py (hunk 1 above)
Comment (philerooski) [reply down_sample_re_count.py]: There's a mistake. for some reason the function is interpreting the `used` and `executed` arguments as annotations.

You're right. I never ended up passing the annotations into the synapseclient.File constructor. I'll fix both problems.

Comment (blogsdon) [reply down_sample_re_count.py]: great, thx

**Review by blogsdon — COMMENTED** [review]
looks good!

**Pull request merged**